onnxruntime/test/providers/cpu/activation/activation_op_test.cc

```diff
@@ -35,12 +35,6 @@ TEST_F(ActivationOpTest, Tanh) {
                          [](float x) { return std::tanh(x); });
 }
 
-TEST_F(ActivationOpTest, Relu) {
```

Comment:
keep as it was?

Relu is an onnx op, not an contrib op.

Reply:
I moved the unit test back here